### DISCUSSION TITLE
Decsion endpoint with cookie-session authenticator.

### DISCUSSION THREAD

**Discussion by rezakazemi928** [ory/oathkeeper#868] [Q&A] [answered]
Hi.
I would like to use the /decisions/ endpoint in the Oathkeeper feature in order to perform the reverse proxy. However, I set the authenticator to cookie-session and I do not know How can I use it on Nginx. Is it even possible to do that??
I test the endpoint and it works perfectly fine on my local machine. I would appreciate it if someone help me with the Nginx part. does it need any extra configuration for the oathkeeper????

**Comment by Benehiko** [comment, marked as answer]
Hi @rezakazemi928 

Not completely sure what you mean, but if you mean passing along the connection from NGINX to Oathkeeper then you can just add the `proxy_pass` option to the NGINX config.

Example:
```
location ~ / {
    proxy_set_header Host $host;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection "upgrade";
    proxy_pass http://127.0.0.1:4455;
  }
```

**Reply by rezakazemi928** [reply]
@Benehiko Thank you so much for your reply.
It was something wrong with my config file. and now everything is working fine.
